joomla/joomla-cms
# [calendar]Fixed translation filenames for sr-* as they would not get loaded due to the capital letters

### Summary of Changes
Fixed translation filenames for sr-* as they would not get loaded due to the capital letters

### Testing Instructions
1) Install `sr-yu` and `sr-rs` languages.
2) Test the calendar with both languages without the patch. The calendar should appear in English.
3) Install patch
4) Test the calendar again with both languages. The calendar should appear in the correct language.

### Documentation Changes Required
Maybe this should be noted somewhere, that the files must be lower-cased.
